### PR TITLE
feat: add Datadog plugin card in plugin hub

### PR DESCRIPTION
**Pull request by guoqqqi**
Fixes: #[Add issue number here]

1. add Datadog plugin card
2. update the icon
image

Screenshots of the change:

### CODE REVIEW

**Comment by guoqqqi** [comment]
cc @bzp2010

**Comment by guoqqqi** [comment]
wait #715

File: website/static/js/plugin-icon.js [hunk 1]
```diff
@@ -31,7 +31,7 @@
...
...-opacity=".69" ></path><path d="M346.744 177.825c3.407-1.128 7.975-1.69 12.544-1.69v206.556c-26.97 2.338-53.28 13.516-73.936 33.535-40.29 39.05-45.856 98.95-16.697 143.796l-64.381 62.398a6.362 6.362 0 0 0-1.932 4.55c0 1.703 0.695 3.338 1.932 4.548l28.244 27.374a6.78 6.78 0 0 0 4.694 1.873 6.78 6.78 0 0 0 4.694-1.873l64.381-62.398c16.264 9.952 34.474 15.73 53 17.335l-0.081 303.11c-3.823 0-7.56-1.129-10.468-3.302L62.383 697.378c-5.732-4.429-10.467-13.768-10.467-20.852V288.129c0-7.086 5.648-14.814 12.544-17.068l282.284-93.234z" fill="#C4C4C4" fill-opacity=".63" ></path><path d="M887.733 259.27a117.42 117.42 0 0 0 14.798-148.236 117.423 117.423 0 1 0-14.798 148.237z m-153.59-83.137a70.457 70.457 0 0 1 70.454-70.452 70.45 70.45 0 0 1 30.06 7.045l-93.938 93.94a70.447 70.447 0 0 1-6.575-30.53z m133.862-30.294a70.464 70.464 0 0 1 7.046 30.296 70.46 70.46 0 0 1-66.159 70.012 70.432 70.432 0 0 1-34.356-6.604l93.469-93.704z" fill="#FF0000" ></path></symbol><symbol id="iconreferer-restriction" viewBox="0 0 1251 1024"><path d="M1131.554 151.59L1244.47 39.206a22.917 22.917 0 0 0 6.476-16.155 22.915 22.915 0 0 0-6.758-16.038 23.122 23.122 0 0 0-16.114-6.726 23.124 23.124 0 0 0-16.228 6.446l-112.9
```
Comment (juzhiyuan) [inline website/static/js/plugin-icon.js:34]: 🤔 May I know how do you generate this file?

File: website/src/pages/plugins.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ function Plugins(props) {
       }
       return (
         <div key={plugin.name}>
-          <PluginCard href={`https://apisix.apache.org/docs/apisix/plugins/${pluginUrl}`} target="_blank">
+          <PluginCard href={`https://apisix.apache.org/docs/apisix/next/plugins/${pluginUrl}`} target="_blank">
```
Comment (juzhiyuan) [inline website/src/pages/plugins.js]: According to mailing list: https://lists.apache.org/thread/fprf95hxjwvbtomndsbd7mh9s7vzfykd

We don't have an agreement on this IMO 🤔

File: website/static/data/plugin.json [hunk 3]
```diff
@@ -145,6 +145,10 @@
             {
                 "name": "zipkin",
                 "description": "Zipkin is a OpenTracing plugin"
+            },
+            {
+                "name": "datadog",
+                "description": "Datadog is a monitoring plugin built into Apache APISIX for seamless integration with Datadog"
```
Comment (juzhiyuan) [inline website/static/data/plugin.json]: Hi @bisakhmondal @yzeng25, do we have a better description for it?

**Comment by juzhiyuan** [comment]
The datatog's plugin link is https://apisix.apache.org/docs/apisix/next/plugins/datadog

@1502shivam-singh Do you have any ideas on this? I'd prefer using this way:

1. Keep the https://apisix.apache.org/plugins/ page using the stable plugins link;
2. Adding a mark or note on Datadog, and saying this plugin is in the Next Version of Apache APISIX.

File: website/static/data/plugin.json (hunk 3 above)
Comment (bisakhmondal) [reply website/static/data/plugin.json]: How about 
> `Datadog` is a monitoring plugin built into Apache APISIX to push metrics of different services into DatadogHQ via a locally run Datadog agent.

Thanks!

**Comment by 1502shivam-singh** [comment]
> 1. Keep the https://apisix.apache.org/plugins/ page using the stable plugins link;
> 2. Adding a mark or note on Datadog, and saying this plugin is in the Next Version of Apache APISIX.

Right, makes sense. We could put a `beta` tag over such plugin cards, like a notification badge, until that particular APISIX version releases.

File: website/static/js/plugin-icon.js (hunk 1 above)
Comment (guoqqqi) [reply website/static/js/plugin-icon.js:34]: I just added an SVG template for it, I haven't found an automated way to generate it yet.
from https://github.com/apache/apisix-website/pull/488

**Comment by guoqqqi** [comment]
I have added a beta tag to the plug-in card and added a comment on the hover, Also updated with links.
If there is still something that needs to be corrected, please help me to point it out. Thanks!
image
image
cc @1502shivam-singh @bzp2010 @bisakhmondal

**Review by juzhiyuan — APPROVED** [review]

**Review by iamayushdas — APPROVED** [review]
Nicely done ✅

**Review by liuxiran — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**